pkg/certify/external-testdriver/driver-call.go

```diff
@@ -0,0 +1,342 @@
+package external_testdriver
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	"github.com/wongma7/csi-certify/pkg/certify/utils"
+	"k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/storage/testpatterns"
+	"k8s.io/kubernetes/test/e2e/storage/testsuites"
+	"os/exec"
+	"strings"
+)
+
+const (
+	getDriverInfo = "getDriverInfo"
+	createVolume  = "createVolume"
+)
+
+var RunCustomTestDriver = true
+var scriptName = ""
+var currentNameSpace = "default"
+var currentDriver *driverDefinition
+
+var validTestDriver = false
+var preprovisionedVolumeTestDriver = false
+var preprovisionedPVTestDriver = false
+
+type DriverCall struct {
+}
+
+type testVolume struct {
+	volumeAttrib map[string]string
+}
+
+var driverCall DriverCall
+
+func init() {
+	flag.Var(&driverCall, "external-testdriver", "name of bashscript that implements all the required testdriver functions")
+}
+
+func (d DriverCall) String() string {
```

Comment:
doesn't seem very useful, but ok, I see driverdef also has a String().

Reply:
flag.Var requires the interface given (driverCall) to implement a Set() and String() function